stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -22,6 +22,22 @@ spec:
         component: {{ include "pipelines.services.name" . }}
       annotations:
         checksum/systemyaml: {{ include (print $.Template.BasePath "/pipelines-system-yaml.yaml") . | sha256sum }}
+        checksum/secretdb: {{ include (print $.Template.BasePath "/database-secret.yaml") . | sha256sum }}
+        {{- if and .Values.buildPlane.dynamic.provider.aws.enabled ( not .Values.buildPlane.dynamic.provider.aws.existingSecret ) }}
+        checksum/secretaws: {{ include (print $.Template.BasePath "/buildplane-secret-aws.yaml") . | sha256sum }}
+        {{- end }}
+        {{- if and .Values.buildPlane.dynamic.provider.aws.enabled }}
```

Comment:
The and is redundant

Reply:
why you think so?

Reply:
Because there's only one condition. It would operate the same without the and, unless I'm missing something

Reply:
we need to operate that one is `aws.enabled` and no external secret is set

Reply:
well but now its only if `aws.enabled` is true. There's no condition for the external secret

Reply:
we need to operate that one is `aws.enabled` and no external secret is set